Magic/pom.xml

```diff
@@ -804,5 +804,40 @@
                 </executions>
             </plugin>
         </plugins>
+        <pluginManagement>
```

Comment:
Please make a separate commit / pr for this

Reply:
Whoops, didn't think that made it into the PR. Reason for that is Eclipse complains about "run" not being a valid goal for the ant plugin if the pluginManagement tag isn't present.

Anyway, I'll remove it.